devel/config-single.yaml

```diff
@@ -41,8 +41,9 @@ network:
 
 logging:
   root:
-    # You might want to use `Trace` and separate output
     level: Info
-    console: true
-    propagate: true
     file: .single/log/root.log
+  agora:
+    level: Trace
+    console: false
+    file: .single/log/agora.log
```

Comment:
This will endlessly happen to a file in a hidden folder... Not a great idea. I'd rather have it log only to the console by default.

Reply:
ok will update